src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,35 @@
 'use strict';
 
+const HEALTH = 100;
+const DAMAGE = 50;
```

Comment:
```suggestion
const DEFAULT_HEALTH = 100;
const HERBIVORE_DAMAGE = 50;
```
Describe your variables better, so everyone can understand to what they are related